dev/service-registry/service-locator/src/main/scala/com/lightbend/lagom/registry/impl/ServiceRegistryActor.scala

```diff
@@ -61,6 +61,10 @@ object InternalRegistry {
           // using "http://" defaults to having no portName and also Some("http") so "http" becomes the
           // default result when searching without a `portName` query.
           Seq(None, Some(serviceUri.getScheme))
+        case "https" =>
+          // using "https://" defaults to having no portName and also Some("https") so "https" becomes the
+          // default result when searching without a `portName` query.
+          Seq(None, Some(serviceUri.getScheme))
```

Comment:
Like for `http`, we should register it twice. Once without a `portName` and on with a `portName`. 

In reality, we have register only without `portName`, but I'm choosing to add both to align with `http`.

Reply:
Review code change done